src/components/filter/MessagesFilterPanel.tsx

```diff
@@ -199,12 +200,28 @@ const MessagesFilterPanel = () => {
 		];
 	}, [messagesStore.messageSessions, sessionsStore.sessions]);
 
+	const updateStreams = (newSteams: string[]) => {
+		if (isFocused) {
+			setStreams(newSteams);
+		} else {
+			searchStore.stopSearch();
```

Comment:
SearchStore is for SearchPanel, i don't get why we should stop search on SearchPanel when sessions on message tab are changed.
I don't understand why we need isFocused state aswell.

Since we refetch messages on sessions change, submit button near sessions input is not longer needed

Reply:
> I don't understand why we need isFocused state aswell.
Filters should be applied after onblur event, that is why I decided use isFocused state.